packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -47,6 +50,8 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
       image: imageProps,
       badge: badgeProps,
       initials: _initials,
+      accessibilityLabel: accessibilityLabel || accessibilityText,
```

Comment:
Should the Avatar and the inner image both have accessibilityLabel and accessibilityHint set if the Avatar is an image?

Also the base Avatar doesn't seem to be set to accessible, should it be?

Reply:
I wasn't sure if we need to add it to the root element. @ksiler , do you know if we need to add it to the root component also?

Reply:
Great question - I hadn't thought about this. I don't think it makes sense to have the Avatar and inner image both have these properties set (especially if no new information is given). I think it makes more sense to have the a11y props on the root element unless giving the user access to any of the inner elements provides extra value/information (from an a11y perspective). So I would probably just place a11yName, a11yHint, accessible, a11yRole, etc. on the root element so when the user lands on it they're given all the information from the base of Avatar. 

In order for the inner elements to not be accessible (so the root is the only thing that is accessed), you'd need to set `accessible={false}` to all the inner elements. I'm trying to test things out on my end but I think some of the inner elements have default accessibility settings that we wouldn't want to conflict with Avatar. I think Icon is also accessible by default and there's no way to change it to false so you may have to edit that.

Reply:
I set icon `accessible` to false but it still was accessible

Reply:
Yeah, I think that's because Icon doesn't give the option to override it. See [Icon.tsx line 23](https://github.com/microsoft/fluentui-react-native/blob/master/packages/experimental/Icon/src/Icon.tsx#L23). It sets `accessible={true}` with no consideration for what's set using the accessible prop like some of our other controls do (like [Checkbox](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Checkbox/src/useCheckbox.ts#L86)).

Reply:
I think it makes sense for it to be able to be overwritten by the parent component, we should modify Icon to allow for accessible to be overridden but default to true otherwise.